chat_shell/chat_shell/tools/builtin/canvas_artifact.py

```
@@ -0,0 +1,491 @@
+    - Managing version numbers
+    """
+
+    name: str = "update_artifact"
+    display_name: str = "更新画布内容"
+    description: str = """Update an existing artifact with new content.
+
+⚠️ CRITICAL: Use this tool IMMEDIATELY when the user asks to modify, edit, change, or update the artifact content.
+
+Common modification requests:
+- "把这句话改成..." (Change this sentence to...)
+- "把XXX改下：...可以改成..." (Change XXX: ... to ...)
```

Ruff (0.14.11)

String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)